app/views/hospitals/_index.html.haml

```diff
@@ -6,7 +6,11 @@
       #list-name= hospital.name
       #list-button-area
         %button.button Remove
-  #list-grid
-    %input.form-input-field{placeholder: "Add Hospitals"}
-    #list-button-area
-      %button.button Add
\ No newline at end of file
+  = form_with url: hospitals_path, scope: :hospital  do |f|
+    #list-grid
+      #list-name
+      = f.text_field :name, id: "Name", class: "form-input-field"
+      #list-button-area
```

Comment:
I still don't understand why this is an id and not a class. (Same with `list-grid` and `list-name`)